front_end/content.py

```diff
@@ -776,27 +794,145 @@ def get_help_request(self, course_id, assignment_id, exercise_id, user_id):
 
         request = self.fetchone(sql, (course_id, assignment_id, exercise_id, user_id,))
         if request:
-            help_request = {"user_id": request["user_id"], "name": request["name"], "code": request["code"], "text_output": request["text_output"], "image_output": request["text_output"], "image_output": request["image_output"], "student_comment": request["student_comment"], "approved": request["approved"], "suggester_id": request["suggester_id"], "approver_id": request["approver_id"], "more_info_needed": request["more_info_needed"]}
+            help_request = {"course_id": course_id, "assignment_id": assignment_id, "exercise_id": exercise_id, "user_id": request["user_id"], "name": request["name"], "code": request["code"], "text_output": request["text_output"], "image_output": request["image_output"], "student_comment": request["student_comment"], "approved": request["approved"], "suggester_id": request["suggester_id"], "approver_id": request["approver_id"], "more_info_needed": request["more_info_needed"]}
             if request["suggestion"]:
                 help_request["suggestion"] = request["suggestion"]
             else:
                 help_request["suggestion"] = None
 
             return help_request
 
+    def compare_help_requests(self, course_id, assignment_id, exercise_id, user_id):
+        #get the original help request, including its output type
+        sql = '''SELECT r.text_output, e.expected_text_output, r.image_output, e.expected_image_output, e.output_type
```

Comment:
Sorry for my poor memory, but does the help_requests table already exist in the database? Or do we need to add it with a migration script?

Reply:
It was added in the 5_to_6 migration script.